content/docs/user-guide/components/reference/terrain/world.md

```diff
@@ -21,10 +21,19 @@ You must add this component to the *Level* entity, the parent of all entities in
 
 | Property | Description | Values | Default |
 | - | - | - | - |
-| World Bounds (min) | The minimum value for the worlds bounds. The **min** value must be less than the **max** value.| Vector3: -2048.0 to 2048.0 | X:`0.0` Y:`0.0` Z:`0.0` |
-| World Bounds (max) | The maximum value for the worlds bounds. The **max** value must be greater than the **min** value. | Vector3: -2048.0 to 2048.0 | X:`1024.0` Y:`1024.0` Z:`1024.0` |
-| Height Query Resolution (m) | The resolution required between each height sample, in meters. | Vector2: 0.001 to Infinity | X:`1.0` Y:`1.0` |
+| Min Height | The minimum value for the height of terrain. The **min** value must be less than the **max** value.| -65536.0 to 65536.0 | 0.0 |
+| Max Height | The maximum value for the worlds bounds. The **max** value must be greater than the **min** value. | -65536.0 to 65536.0 | 1024.0 |
+| Height Query Resolution (m) | The distance between each height sample position, in meters. | 0.1 to Infinity | 1.0 |
+| Surface Data Query Resolution (m) | The distance between each surface data sample position, in meters. | 0.1 to Infinity | 1.0 |
 
-{{< note >}}
-Because large numbers of terrain height samples can take a long time to calculate, O3DE will not allow the number of samples to exceed 8 million.
-{{< /note >}}
+
+## Usage 
+
+The **Min Height** and **Max Height** bounds should be set to as small a range as possible for the most accurate height values. For instance, the terrain renderer uses 16-bit height values, which provides 65536 possible distinct heights. If the difference between the min and max height is 1 km, that's a resolution of about 1.5 cm.
+
+**Height Query Resolution** and **Surface Data Query Resolution** are synchronization points for terrain height and surface data used by different terrain systems like physics and rendering. They effectively create grids across the world - one for height, and one for surface data. All height and surface data providers will be queried at this grid resolution regardless of the resolution of the data in providers. Ideally, source data in providers with quantized data (like image gradients) will match the query resolution used.
```

Comment:
What is meant by "synchronization points" and "providers"? "Providers" may be ambiguous. 

Similarly, "source data" may be ambiguous, so I updated the last sentence. Can you verify if that's right?

```suggestion
 **Height Query Resolution** and **Surface Data Query Resolution** are synchronization points for both terrain height and surface data. Different terrain systems, like physics and rendering, use them to create height and surface data grids across the world. All height and surface data providers are queried at this grid resolution, regardless of the resolution of the data in providers.  
 
We recommend that heightmaps or surface data that have quantized source data, like image gradients, match the Height Query Resolution.
```

Reply:
"Synchronization points" - The query resolutions tell other systems that rely on terrain how to query data for their system. I referred to them as synchronization points because it helps keep different terrain systems in sync. As an example, if the query resolution is 1m, then both rendering and physics will query the same points 1 meter apart for their data, and everything will like up - a ball will bounce off the terrain in the location that it looks like it should bounce. Without this property, we'd have to leave it up to the user to make sure they made everything sync up by themselves. So, they don't create actual grids across the world, it's just a way of understanding how they work (ie query in regular intervals in x and y according to the query resolution)

"Providers" - this just means any component which can respond to queries about heights or surface data - it "provides" data for the terrain system.

"Source data" I think your clarification works. Source data could provide to anything providing the height or surface information like an image or noise function.

Reply:
How about this:

> **Height Query Resolution** and **Surface Data Query Resolution** are synchronization points for both terrain height and surface data. All of the entities providing input height and surface data for the terrain system will be queried at this grid resolution regardless of the resolution of the data contained in those entities. We recommend that heightmaps or surface data that have quantized source data, like image gradients, match the Height Query Resolution.

Reply:
This actually provoked a long discussion with @mbalfour-amzn and this is what we landed on:

> **Height Query Resolution** and **Surface Data Query Resolution** are used to define consistent spacing, centered at the origin, for terrain height and surface queries across the entire world. Different systems that use terrain data, such as physics and rendering, can also use these conceptual grids to make implementation decisions about how to best use the terrain data. The input data to the terrain system can be authored or generated at any resolution, but the terrain system will by default query the input data only at points aligned to the grid. This creates a consistent view of the input data. Since the input data will normally only be used at these resolutions, we recommend that input gradients that have quantized source data, like images, match the appropriate query resolution.

Reply:
Great! This makes a lot more sense. And "conceptual grid" is a good way to describe it.

```suggestion
**Height Query Resolution** and **Surface Data Query Resolution** are used to define consistent spacing, centered at the origin, for terrain height and surface queries across the entire world. Different systems that use terrain data, such as physics and rendering, can also use these conceptual grids to make implementation decisions about how to best use the terrain data. By default, the terrain system queries the input data only at points aligned to the grid. However, you can author or generate input data to the terrain system at any resolution. This creates a consistent view of the input data. Since the input data is only used at these resolutions, we recommend that input gradients that have quantized source data, like images, match the appropriate query resolution.
```